### ISSUE TITLE
PathWithPrefixFlowAttribute wrong parameters for get()

### ISSUE THREAD

**Issue by mattiadg** [rwth-i6/i6_core#66]
I've come into `PathWithPrefixFlowAttribute` in https://github.com/rwth-i6/i6_core/blob/main/rasr/flow.py because of an error I'm getting in `EstimateLDAMatrixJob`.
Here, the `get` method requires a `net` argument that is not used and returns `path` instead of `self.path`.
Also, do we really want to call `get` explicitly to get a string representation of it?

**Comment by JackTemaki** [comment]
Can you specify the full error, e.g. the Stacktrace?

**Comment by JackTemaki** [comment]
Ah sorry, you do not have to, it is clear what is wrong. Let me check the old version to see what happened here...

**Comment by mattiadg** [comment]
in the rasr config I get the object representation instead of a string
e.g. `between-class-scatter-matrix-file = <i6_core.rasr.flow.PathWithPrefixFlowAttribute object at 0x7f7b674359e8>`

**Comment by mattiadg** [comment]
Maybe `get(self, net)` is part of the interface of Flow objects, but at least `self.path` should be fixed.

**Comment by michelwi** [comment]
> Let me check the old version to see what happened here...

in the old version we had
```
    if cls.FIX_MATRIX_PATH_BUG:
      config.acoustic_model_trainer.lda_estimator.between_class_scatter_matrix_file = sprint.PathWithPrefixFlowAttribute('xml:', between_class_scatter_matrix)
      config.acoustic_model_trainer.lda_estimator.within_class_scatter_matrix_file  = sprint.PathWithPrefixFlowAttribute('xml:', within_class_scatter_matrix)
    else:
      config.acoustic_model_trainer.lda_estimator.between_class_scatter_matrix_file = 'xml:%s' % str(between_class_scatter_matrix)
      config.acoustic_model_trainer.lda_estimator.within_class_scatter_matrix_file  = 'xml:%s' % str(within_class_scatter_matrix)
```

and I guess the fix was not tested.


> but at least `self.path` should be fixed. 

I agree.

**Comment by JackTemaki** [comment]
> and I guess the fix was not tested.

I checked the code, this never could have worked.

I will figure out together with Mattia how to fix this

**Comment by mattiadg** [comment]
This issue is only about `PathWithPrefixFlowAttribute`. If `net` is a needed argument, though unused, I think we can close this

**Comment by curufinwe** [comment]
Closed by #69
